api/client/client.go

```diff
@@ -266,3 +276,21 @@ func NewAuthServer(addr string, provider auth.Provider) *AuthRemote {
 func (ar *AuthRemote) Sign(req []byte) ([]byte, error) {
 	return ar.AuthSign(req, nil, ar.provider)
 }
+
+// nomalizeURL checks for http/https protocol, appends "http" as default protocol if not defiend in url
+func normalizeURL(addr string) string {
+	addr = strings.TrimSpace(addr)
+	u, err := url.Parse(addr)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if !strings.Contains(u.Scheme, "http") {
```

Comment:
i used string because take for example u = 127.0.0.1:8888, u.Scheme would be "127.0.0.1" not  " ".